### PR TITLE
Fixes #27872 - Use delete instead of remove on ch removal

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/i18n/bastion_katello.pot [hunk 1]
```diff
@@ -4313,7 +4313,7 @@ msgid "Remove Host Collection \"{{ hostCollection.name }}\"?"
 msgstr ""
 
 #: app/assets/javascripts/bastion_katello/content-hosts/views/content-hosts.html
-msgid "Remove Hosts"
+msgid "Destroy Hosts"
```
Comment (jturel) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/i18n/bastion_katello.pot]: I don't think this file needs to be changed. it's auto-generated afaik

**Comment by jturel** [comment]
The BZ mentioned using the word Delete rather than Destroy - was this an intentional distinction?

**Comment by chris1984** [comment]
@jturel was this in content-hosts or all-hosts. The content-hosts is covered in this pr, but if you are seeing it in all-hosts I think I will prob have to make a PR in Foreman for that part.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-destroy-modal.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
 <div data-extend-template="components/views/bst-modal.html">
   <h4 data-block="modal-header" translate>Remove {{ table.numSelected  }} Hosts?</h4>
```
Comment (jturel) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-destroy-modal.html]: s/Remove/Delete

Comment (jturel) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-destroy-modal.html]: The 'Remove' button is also in this modal

**Comment by chris1984** [comment]
@jturel updated, showing delete in both spots now:

https://nimb.ws/4CHGzo

**Comment by jturel** [comment]
[test katello]

**Comment by chris1984** [comment]
@jturel green tests :) :beers:

**Pull request merged**